src/utils/Common.ts

```diff
@@ -350,4 +350,6 @@ export function assign<A extends B, B extends Object>(source: A, assignments: B)
 const baseConfigDirectory = Path.join(homeDirectory, ".black-screen");
 export const presentWorkingDirectoryFilePath = Path.join(baseConfigDirectory, "presentWorkingDirectory");
 export const historyFilePath = Path.join(baseConfigDirectory, "history");
+// We can't use HISTFILE to get the path, so just check the default location
+export const bashHistoryFilePath = Path.join(homeDirectory, ".bash_history");
```

Comment:
There is a better place for that. https://github.com/shockone/black-screen/blob/master/src/utils/Shell.ts. You can also move deserialization logic there, if it differs across shells.